po/com.github.cassidyjames.ephemeral.pot

```diff
@@ -269,59 +279,63 @@ msgid ""
 "but they won’t stop your ISP, government, or determined websites from "
 "tracking you."
 msgstr ""
+"Unutmayın, Geçici ve herhangi bir tarayıcının gizli veya özel modu çok "
+"şey yapabilir: bazı izlemeleri hafifletir ve cihazınızda veri depolamazlar, "
+"ancak ISS'nizin, devletinizin veya kararlı web sitelerinizin sizi "
+"izlemesini durdurmazlar."
 
 #: src/Views/WelcomeView.vala:34
 msgid "For the best protection, always use a VPN."
-msgstr ""
+msgstr "En iyi koruma için her zaman bir VPN kullanın."
 
 #. TRANSLATORS: Includes an ellipsis (…) in English to signify the action will be performed in another menu
 #: src/Widgets/BrowserButton.vala:50
 msgid "Open page in…"
-msgstr ""
+msgstr "Sayfayı şurada aç…"
 
 #: src/Widgets/BrowserButton.vala:62
 msgid "Close Window When Opening Externally"
-msgstr ""
+msgstr "Harici Olarak Açarken Pencereyi Kapat"
 
 #: src/Widgets/BrowserButton.vala:141
 #, c-format
 msgid "Open page in %s"
-msgstr ""
+msgstr "Sayfayı %s içinde aç"
 
 #: src/Widgets/UrlEntry.vala:39
 msgid "Enter a URL or search term"
-msgstr ""
+msgstr "Bir URL veya arama terimi girin"
 
 #: src/Widgets/UrlEntry.vala:185
 #, c-format
 msgid "Go to \"%s\""
-msgstr ""
+msgstr "\"%s\ Adresine git"
 
 #: src/Widgets/UrlEntry.vala:185
 #, c-format
 msgid "Search for \"%s\""
-msgstr ""
+msgstr "\"%s\ Şunu Ara"
```

Comment:
This is a quoted string, so it should probably be:

```suggestion
msgstr "\"%s\" Şunu Ara"
```

Reply:
I'm fixing now